modules/db/src/test/java/org/mule/module/db/internal/resolver/database/StaticDbConfigResolverTestCase.java

```diff
@@ -72,4 +78,15 @@ public void returnsMetaData() throws Exception
 
         assertThat(metaData, is(expectedMetaData));
     }
+
+    @Test
+    public void disposesDisposableDbConfig() throws Exception
+    {
+        DataSource dataSource = mock(DataSource.class, withSettings().extraInterfaces(Disposable.class));
+        when(dbConfig.getDataSource()).thenReturn(dataSource);
+
+        ((StaticDbConfigResolver) dbConfigResolver).dispose();
```

Comment:
dbConfigResolver should be defined as StaticDbConfigResolver instead of using a cast